generators/server/templates/src/main/java/package/config/OAuth2Configuration.java.ejs

```diff
@@ -16,10 +18,21 @@ public class OAuth2Configuration {
         <%= reactivePrefix %>ClientRegistrationRepository clientRegistrationRepository,
         <%= reactive ? 'Server' : '' %>OAuth2AuthorizedClientRepository authorizedClientRepository
     ) {
-        <%= reactivePrefix %>OAuth2AuthorizedClientManager authorizedClientManager = new Default<%= reactivePrefix %>OAuth2AuthorizedClientManager(
+        Default<%= reactivePrefix %>OAuth2AuthorizedClientManager authorizedClientManager = new Default<%= reactivePrefix %>OAuth2AuthorizedClientManager(
             clientRegistrationRepository,
             authorizedClientRepository
         );
+
+        authorizedClientManager.setAuthorizedClientProvider(
+            <%= reactivePrefix %>OAuth2AuthorizedClientProviderBuilder
+                .builder()
+                .authorizationCode()
+                .refreshToken(builder -> builder.clockSkew(Duration.ofMinutes(1)))
+                .clientCredentials()
+                .password()
```

Comment:
This looks strange. Why do we need a password here and where does it come from?

@bdemers Can you have a look at this? We're trying to get refresh tokens working in JHipster.

Reply:
It's the default for the DefaultOAuth2AuthorizedClientManager
https://github.com/spring-projects/spring-security/blob/006b9b960797d279b31cf8c8d16f1549c5632b2c/oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/DefaultOAuth2AuthorizedClientManager.java#L91-L96

I've added to let the user easily change clockSkew, but can be removed.